test/models/user_test.rb

```diff
@@ -201,4 +201,20 @@ class UserTest < ActiveSupport::TestCase
     user.dismiss_introducer!('scratchpad')
     assert user.introducer_dismissed?('scratchpad')
   end
+
+  test "email is sent after confirmation" do
+    user = create :user
+
+    User::Notification::CreateEmailOnly.expects(:call).with(user, :joined_exercism, {})
+
+    user.confirm
+  end
+
+  test "email is sent if confirmation is skipped" do
```

Comment:
Maybe be slightly more specific?

```suggestion
  test "joined exercism email is sent if confirmation is skipped" do
```